### ISSUE TITLE
`unit-test-without-phi` Linter Mishandles Nested Unit Tests in EO `0.56.0`

### ISSUE THREAD

**Issue by maxonfjvipon** [objectionary/lints#515]
In EO `0.56.0` we've changed the structure of unit tests:
```
# Test.
[] > stdout-tests
  [] > test-works
    stdout > @
      "Hello, world"
```

But I see this: `org.eolang.io.stdout-tests:12 The unit test doesn't have "@" attribute: "stdout-tests" (unit-test-without-phi)`.

We should change the behaviour of the lint so it checks nested tests of top level object

**Comment by 0crat** [comment]
@maxonfjvipon Hello! Thank you for reporting a new bug. Your contribution is valuable in improving our project's quality. As per our team's policy, you've earned **8** points for this report. Your current balance is now [+112](https://www.eolang.org/zerocracy/objectionary-vitals.html). We encourage you to continue [reporting](https://www.yegor256.com/2018/04/24/right-way-to-report-bugs.html) any issues you find. If you notice anything that [doesn't look](https://www.yegor256.com/2018/02/06/where-to-find-more-bugs.html) right, please [don't hesitate](https://www.yegor256.com/2014/04/13/bugs-are-welcome.html) to let us know. Your vigilance helps maintain the integrity of our repository. Keep up the great work!

**Comment by h1alexbel** [comment]
@maxonfjvipon please try new version: [0.0.48](https://github.com/objectionary/lints/releases/tag/0.0.48). It should solve the issue

**Comment by VasilevNStas** [comment]
@maxonfjvipon @volodya-lombrozo, this one is **already fixed on `master`** (and was shipped in 0.0.48 per the thread). The nested-unit-test handling is implemented in commit `8814547c` (`bug(#515): simplify 2x`) and covered by pack tests that are green in the current build:
- `packs/single/unit-test-without-phi/catches-only-nested-test-object-without-phi.yaml`
- `packs/single/unit-test-without-phi/catches-test-with-phi-under-nested-abstract.yaml`

The nested `[] > stdout-tests / [] > test-works` structure from the report no longer triggers a false positive on the top-level object. Suggest closing.

**Comment by volodya-lombrozo** [comment]
Confirmed fixed on `master`: commit `8814547c0` ("bug(#515): simplify 2x") resolves this and is merged into master. Regression tests cover the exact nested-unit-test scenario from this report:
- `packs/single/unit-test-without-phi/catches-only-nested-test-object-without-phi.yaml`
- `packs/single/unit-test-without-phi/catches-test-with-phi-under-nested-abstract.yaml`

Also confirmed by @h1alexbel (shipped in 0.0.48). Closing.

**Comment by 0crat** [comment]
@h1alexbel Thanks for closing this issue! You've earned +4 points for this: +8 as a basis; -20 for 473 days of delay; +16 to give you at least something. Your running score is [+85](https://www.eolang.org/zerocracy/objectionary-vitals.html); don't forget to check your [Zerocracy account](https://www.zerocracy.com/) too).
